spec/decorators/article_decorator_spec.rb

```diff
@@ -57,7 +57,12 @@ def create_article(*args)
     it "creates proper description when it is not present and body is not present and long, and tags are present" do
       body_markdown = "---\ntitle: Title\npublished: false\ndescription:\ntags: heytag\n---\n\n"
       created_article = create_article(body_markdown: body_markdown)
-      expect(created_article.description_and_tags).to eq("A post by #{created_article.user.name.delete('.')}. Tagged with heytag.")
+      parsed_post_by_string = if created_article.user.name.end_with?(".")
```

Comment:
This is probably a nitpick/style preference so I'm approving this and leaving this as a suggestion.

What about:
```ruby
parsed_post_by_string = "A post by #{created_article.user.name}"
parsed_post_by_string += "." unless created_article.user.name.end_with?(".")
```
Only because I had to look twice. At first, I thought the conditions were the same by accident and then I noticed the period.

Reply:
that actually matches really nicely with the way it is written in the decorator +1